script/node_bs.py

```diff
@@ -0,0 +1,281 @@
+from distutils.log import ERROR, INFO, WARN
+from logging import WARNING
+
+import os
+import json
+import requests
+import subprocess
+from subprocess import call
+import glob
+import stat
+import docker
+import shutil
+import sys
+from datetime import date
+
+
+GITHUB_PACKAGE_INFO_API = "https://api.github.com/repos/{}/{}/{}/{}"
+
+
+path_separator = os.path.sep
+#ROOT = os.path.dirname(os.path.dirname(__file__))
+ROOT = os.getcwd()
+package_name = input("Enter Package name (Package name should match with the directory name): ")
+#package_name = 'elasticsearch'
+package_name = package_name.lower()
+dir_name = f"{ROOT}{path_separator}{package_name[0]}{path_separator}{package_name}"
+
+
+github_url=''
+latest_release=''
+active_repo=False
+new_build_script=''
+branch_pkg=""
+
+
+def get_latest_build_script(dir_name):
+    files_path = os.path.join(dir_name,"*")
+    files = sorted(glob.iglob(files_path),key=os.path.getctime,reverse=True)
+    latest_existing_build_script =""
+    for script_file in files:
+        if script_file.endswith(".sh") and (script_file.find("ubuntu")==-1):
```

Comment:
Just a suggestion, this line can also be written as:
`if script_file.endswith(".sh") and script_file.find("ubuntu") < 0:`